app/routes.py

```diff
@@ -1 +1,302 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, make_response, request, jsonify, abort
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from sqlalchemy import asc, desc
+from datetime import datetime, date
+import requests
+import json
+# For Slack Bot
+import os
+from dotenv import load_dotenv
+load_dotenv()
+token = os.environ.get("SLACK_BOT_TOKEN")
+
+
+# Blueprints
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+# Helper function: validate model
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{type(model_id)} is not a valid type"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} #{model_id} not found"}, 404))
+    
+    return model
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    try:
+        request_body = request.get_json()
+        new_task = Task(
+            title = request_body["title"],
+            description =  request_body["description"],
+            # completed_at = request_body["completed_at"]
+        )
+
+        db.session.add(new_task)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model